packages/website/sidebars/sidebar.base.js

```diff
@@ -41,6 +43,7 @@ module.exports = {
         {
           label: 'Architecture',
           type: 'category',
+          collapsed: false,
```

Comment:
What do you think about switching these to:

```suggestion
          collapsible: false,
```

Reasoning: since there are so few menu options, I can't think of any reason why someone would want to collapse them. So the buttons to toggle collapse them are unnecessary UI.

Reply:
Yeah makes sense, so no collapsible pane at all. 

Let me try it.

Reply:
![image](https://user-images.githubusercontent.com/11923975/188722748-b4820244-ab7f-4899-ac05-60b25b30cd5b.png)
this is how it looks after this